packages/muelu/test/maxwell/Maxwell3D.cpp

```diff
@@ -72,6 +72,7 @@
 #include <BelosTpetraAdapter.hpp>
 #include <BelosXpetraAdapter.hpp>     // => This header defines Belos::XpetraOp
 //#include <BelosMueLuAdapter.hpp>      // => This header defines Belos::MueLuOp
+#include <BelosBlockCGSolMgr.hpp>
```

Comment:
What's this for?

Reply:
Changed this to call the registerCustomSolverFactory but needs further discussion. Please see general note below on this call - this can be automatic but then i think we'll have to put the includes back in the SolverFactory header for all the managers.